src/handlers/settlement-windows.js

```diff
@@ -19,12 +19,17 @@ const handler = (router, routesContext) => {
     });
 
     router.get('/settlement-windows/:settlementWindowId', async (ctx, next) => {
-        const api = new Model({ endpoint: routesContext.config.settlementsEndpoint });
-        const [settlementWindow, settlement] = await Promise.all([
-            routesContext.db.getSettlementWindowInfo(ctx.params.settlementWindowId),
-            api.getSettlements({ settlementWindowId: ctx.params.settlementWindowId }),
```

Comment:
Aaron, what was this returning previously? Was this the source of the issue?

Reply:
This is still running when the window is settled. The problem was that the api will return a 3001 error code when settlements for an open Window are requested. since an open Window cannot have settlements yet, this causes some problems with the portal.